gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Comment:
@imuchnik is `4.1` the correct version number to use? I ask because that version is almost a year old, and it seems we should update to the latest if possible. What do you think?

Reply:
We can potentially go  to 4.2, but starting at 4.2.1 there are breaking changes that will not work without some more time spent on code.